producers.md

```diff
@@ -530,6 +551,8 @@ Since all tests output TAP, they can be run with any TAP harness.
 **[Tapper-autoreport](https://github.com/tapper/Tapper-autoreport)**
 turns bash scripts into TAP test scripts.
 
+**[uze TAP](https://github.com/zsh-uze/devel-tap)** is a [zsh](http://www.zsh.org) provide helpers similar to the [Test2 suite](https://metacpan.org/release/Test2-Suite). This module is a part of the [uze](http://zsh-uze.github.io/) ecosystem.
```

Comment:
Is there a word missing in this sentence? "is a zsh provide helpers" doesn't read like proper English to me. Am I missing something?

Reply:
"library that" is missing... i'll fix